desc/basis.py

```diff
@@ -261,9 +261,9 @@ def _get_modes(self, L):
             Each row is one basis function with modes (l,m,n).
 
         """
-        l = np.arange(L + 1).reshape((-1, 1))
-        z = np.zeros((L + 1, 2))
-        return np.hstack([l, z])
+        l = np.arange(L + 1)
+        z = np.zeros_like(l)
+        return np.array([l, z, z]).T
```

Comment:
I changed some of the mode number constructors. They are the same as before. I changed this while debugging github issue 1522 and decided to keep the modifications because it uses the simpler conventions we use elsewhere in the code.